broadcast.go

```diff
@@ -86,6 +86,7 @@ func (c *nopBroadcaster) SendAsync(pb proto.Message) error {
 // handle broadcast messages. (Hint: this is implemented by pilosa.Server)
 type BroadcastHandler interface {
 	ReceiveMessage(pb proto.Message) error
+	SendSync(pb proto.Message) error
```

Comment:
was it necessary to add `SendSync` to the `BroadcastHandler` interface? Why not just keep it part of the `Broadcaster` interface?

Reply:
`func (g *GossipNodeSet) SendSync(pb proto.Message) error` does not have access to the Cluster Node list.  Seemed easier to pass this call down the the `Server` implementation of `BroadcastHandler ` to manage that with the already existing cluster info plus a default client.